streaming_form_data/targets.py

```diff
@@ -35,20 +39,47 @@ def value(self):
 
 
 class FileTarget(BaseTarget):
-    def __init__(self, filename):
+    def __init__(self, filename, openmode='wb'):
```

Comment:
I want to have an option to switch to "xb". "xb" will prevent to create already existing file.

This is the only reason. So it could be some boolean argument instead. Lets call it "allow_overwrite_file" with default value True.

Reply:
Sounds good. Let's call it `allow_overwrite` for brevity.